### PR TITLE
GroupImportFixer - Fix failing case

### PR DESCRIPTION
**Pull request by julienfalque**

### CODE REVIEW

File: tests/Fixer/Import/GroupImportFixerTest.php [hunk 1]
```diff
@@ -291,25 +306,41 @@ public function provideFixCases()
...
 use function Foo\baz;
 use Foo\Baz;
 ',
-            ],
         ];
 
-        foreach ($tests as $index => $test) {
-            yield $index => $test;
-        }
+        yield [
```
Comment (julienfalque) [inline tests/Fixer/Import/GroupImportFixerTest.php:325]: Sorry for the noise, I replaced the array with `yield` statements. The failing case is added here.

Comment (keradus) [reply tests/Fixer/Import/GroupImportFixerTest.php:325]: not only noise, but also conflicts... :(

**Comment by keradus** [comment]
Thank you @julienfalque.

**Comment by julienfalque** [comment]
Thank you @keradus!

**Pull request merged**